Fake-News-Detection/result.json

```diff
@@ -0,0 +1,18 @@
+{
+    "Title": "Fake-News-Detection",
+    "Tags": ["Natural Language Processing"],
+    "Architecture": "Logistic Regression",
+    "Publisher": [["Geeky-star","https://github.com/Geeky-star"], ["Smoketrees","https://github.com/smoke-trees"]], 
```

Comment:
As you aren't a part of the SmokeTrees organization, please remove it from the name of publishers